gce/sysprep/instance_setup.ps1

```diff
@@ -288,11 +288,15 @@ function Configure-WinRM {
     # https://msdn.microsoft.com/en-us/library/windows/desktop/aa386968(v=vs.85).aspx
     $eku = '1.3.6.1.5.5.7.3.1,1.3.6.1.5.5.7.3.2'
     & $script:gce_install_dir\tools\makecert.exe -r -a SHA1 -sk "$(hostname)" -n "CN=$(hostname)" -ss My -sr LocalMachine -eku $eku
-    $cert = Get-ChildItem Cert:\LocalMachine\my | Where-Object {$_.Subject -eq "CN=$(hostname)"}
+    $cert = Get-ChildItem Cert:\LocalMachine\my | Where-Object {$_.Subject -eq "CN=$(hostname)"} | Select-Object -First 1
   }
   # Configure winrm HTTPS transport using the created cert.
   $config = '@{Hostname="'+ $(hostname) + '";CertificateThumbprint="' + $cert.Thumbprint + '";port="5986"}'
-  _RunExternalCMD winrm create winrm/config/listener?Address=*+Transport=HTTPS $config
+  _RunExternalCMD winrm create winrm/config/listener?Address=*+Transport=HTTPS $config -ErrorAction SilentlyContinue
```

Comment:
In case of an error, this will print an exception to the console output. I don't know if you want that to happen, but it seems suboptimal for expected behavior.